source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -305,7 +310,12 @@ bool FragmentedRequest::onChildRedirection(const Common::Redis::RespValue& value
 
   this->pending_requests_[index].handle_ =
       conn_pool->makeRequestToHost(host_address, request, this->pending_requests_[index]);
-  return (this->pending_requests_[index].handle_ != nullptr);
+
+  if (this->pending_requests_[index].handle_ != nullptr) {
```

Comment:
nit: remove `this` here and above as long as you are in here.

Reply:
ok